pkg/bootstrap/node.go

```diff
@@ -67,7 +67,7 @@ func NewNode(baseImage, machine string) error {
 	return nil
 }
 
-func NodeExists(machine string) bool {
+func IsMachineImageExists(machine string) bool {
```

Comment:
Should this not be `MachineImageExists`? Don't get why we need to the `Is`. Doesn't read right.